### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD022.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#27] [General]
# https://blog.3place.cn/3place/fsd/FSD022.html

https://blog.3place.cn/3place/fsd/FSD022.html

**Comment by picklecai** [comment]
长明灯的光向着上方照射，
而向侧面却逐渐减弱，
黑暗在四周团团进逼，
我的心就如此被黑夜笼罩。

浮士德的这个感受从何而来？

**Comment by picklecai** [comment]
即使天主能对你恕罪，
你也得要不齿于人类！

逃过天谴，也会人言可畏。

**Comment by picklecai** [comment]
先生，莫后退！振作起来！
紧跟着我，听我的话。
快把拂尘拔了出来！
只管刺去！我来招架。

所谓的魔鬼附体，原来是这样
